docs/pages/getting_started_access_data.mdx

```diff
@@ -0,0 +1,89 @@
+
+**Namespace Prefixes:**
+Each origin supports one or more *namespace prefixes*, which are analogous to the folders or directories from your computer that you use to organize files. Below is an example object:
+```bash
+/demo/testfile.txt
+```
+This is a simple example where the namespace prefix would be `/demo` and the object is named `/testfile.txt`.
+
+However, things can get a little tricky when you add a longer namespace prefix and/or a longer filepath within that namespace where the object lives. For example:
+```bash
+/ospool/uc-shared/public/OSG-Staff/validation/test.txt
+```
+In this example, the namespace prefix is `/ospool/uc-shared/public/`, and the actual object is named `/OSG-Staff/validation/test.txt`.
+
+>**How to differentiate namespace prefix and object name?**
+To discover where the namespace prefix ends and the object name begins, there are a few options. One way is to access your federation's registry (i.e. [https://osdf-registry.osg-htc.org/view/registry/](https://osdf-registry.osg-htc.org/view/registry/)) which should list the namespaces available in the federation. Another option is also asking your federation administrator for how to access the registry. However, you can always think to just combine these two terms into just the "object's path" or the "path to the object within the federation".
+
+**Federations:**
+Objects in Pelican belong to *federations*, which are aggregations of data that are exposed to other individuals in the federation. Each Pelican federation constitutes its own global namespace of objects and each object within a federation has its own path, much like files on a computer. Fetching any object from a federation requires at minimum two pieces of information: the federation's root (i.e. the discovery URL), and the path to the object within that federation (there is the potential that some objects require access tokens as well, but more on that later). For example, the OSDF’s federation hostname is osg-htc.org, an example path to an object in the federation is:
+```bash
+/ospool/uc-shared/public/OSG-Staff/validation/test.txt
+```
+
+Combining these two pieces of information, we say the the **object's URL** is:
+```bash
+pelican://osg-htc.org/ospool/uc-shared/public/OSG-Staff/validation/test.txt
+```
+
+## Your First Pelican Client Command
+You can easily access objects from a Pelican federation with the use of the Pelican client commands. Here we will be using the `pelican object get` command to get an object from a federation. This command is formatted like so:
```

Comment:
```suggestion
You can easily access public objects from a Pelican federation with the use of the Pelican client commands. Here we will be using the `pelican object get` command to get an object from a federation. This command is formatted like so:
```

To get protected objects requires tokens, which I hesitate to describe as "easy".